server/channels/app/support_packet_test.go

```diff
@@ -394,3 +397,20 @@ func TestCreateSanitizedConfigFile(t *testing.T) {
 	assert.Positive(t, len(fileData.Body))
 	assert.NoError(t, err)
 }
+
+func TestCreateSupportPacketMetadata(t *testing.T) {
```

Comment:
nit: maybe add test to cover error in metadata generation and error in YAML marshalling?

Reply:
I did find it hard to add a test for the error case as `createSupportPacketMetadata` only calls `model.GeneratePacketMetadata`. Surely, it's possible to introduce dependency injection and replace the implementation of `model.GeneratePacketMetadata` with something else, but I don't think that added complexity is worth the additional test coverage.

Do you have other suggestions on how to test the error case?